apps/zotonic_mod_acl_user_groups/src/support/acl_user_groups_checks.erl

```diff
@@ -202,6 +202,14 @@ acl_is_allowed(_, #context{user_id=1}) ->
     true;
 acl_is_allowed(#acl_is_allowed{object=undefined}, _Context) ->
     undefined;
+acl_is_allowed(#acl_is_allowed{action=view, object=Id}, #context{ user_id = UserId })
```

Comment:
```suggestion
acl_is_allowed(#acl_is_allowed{ action=view, object=Id }, #context{ user_id = UserId })
```

Reply:
Maybe we should re-do all clauses in this file, to make them consistent.

Reply:
Yes, maybe also make sense to discuss plugins for formatting like https://github.com/AdRoll/rebar3_format

Reply:
Yes, that would be a good thing to check and then have a one-time big update.